try:
    print(x)
except:
    print("something went wrong")
finally:
    print("Success")